http/file_server.ts

```diff
@@ -830,7 +830,13 @@ function main() {
   const useTls = !!(keyFile && certFile);
 
   function onListen({ port, hostname }: { port: number; hostname: string }) {
-    const networkAddress = getNetworkAddress();
+    let networkAddress: string | undefined = undefined;
+    if (
+      Deno.permissions.querySync({ name: "sys", kind: "networkInterfaces" })
+        .state === "granted"
```

Comment:
I think this will fail in Deno Deploy – proposal to fix in #6033

Reply:
Good catch. Thanks!